line_profiler.py

```diff
@@ -369,7 +369,11 @@ def magic_lprun(self, parameter_s=''):
 def load_ipython_extension(ip):
     """ API for IPython to recognize this module as an IPython extension.
     """
-    ip.define_magic('lprun', magic_lprun)
+    if hasattr(ip, 'define_magic'):
```

Comment:
I got an `AttributeError: 'ZMQInteractiveShell' object has no attribute 'define_magic'` on this line.